frontend/src/components/BackButton.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+
+const BackButton = () => {
+  const containerStyle = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  };
+
+  const buttonStyle = {
+    padding: "10px 20px",
+    fontSize: "16px",
+    borderRadius: "5px",
+    backgroundColor: "#0d37de",
+    color: "#fff",
+    textDecoration: "none",
+  };
+
+  return (
+    <div style={containerStyle}>
+      <Link to="/" style={{ textDecoration: "none" }}>
+        <button type="button" style={buttonStyle}>
+          Go back
```

Comment:
Could we potentially give this a more descriptive label?

In the wireframes "Back to Scenarios" were used - this has the additional benefit of clearly telling the user where they will be navigated.